common/lib/common-utils/src/test/mocha/idleTaskScheduler.spec.ts

```diff
@@ -4,14 +4,19 @@
  */
 
 import { strict as assert } from "assert";
+
 import { SinonFakeTimers, useFakeTimers, fake, match } from "sinon";
+
 import * as idleTask from "../../idleTaskScheduler";
 
 describe("Idle task scheduler", () => {
 	let clock: SinonFakeTimers;
 
-	beforeEach("setClock", () => {
+	before(() => {
```

Comment:
Should `clock` just be assigned on line 13 and skip the hook altogether?

Reply:
Possibly - I elected to make the minimal change since I was focused on dependency updates.

If we wanted to invest more here, I'm not exactly on board with the polyfill as currently written.  In particular I think (at least for general use) the fallback should be to immediately invoke the callback rather than waiting the maximum timeout (as there's no guarantee or expectation that the system will be more idle at the time of the timeout).

I also see that it's marked deprecated and not currently used anywhere, so it might also make sense to just remove.